### PR TITLE
[Sage-511] Sage 3 - Loader Update

### PR DESCRIPTION
**Pull request by anechol**
## Description
- Updates the height of the loader bar from 10px to 6px.
- Add new "success" variant.

## Screenshots

|  Before  |  After  |
|--------|--------|
Screen Shot 2021-08-20 at 1 46 29 PM|Screen Shot 2021-08-20 at 1 43 40 PM

Screen Shot 2021-08-23 at 10 12 50 AM

## Testing in `sage-lib`

- View [Loader](http://localhost:4000/pages/component/loader) component
- Check that bar loader height has been updated
- Check that "success" variant exists and isn't broken

## Testing in `kajabi-products`

LOW. Updates progress bar height and add new "success" variant.

## Related
#511

### CODE REVIEW

File: packages/sage-assets/lib/stylesheets/components/_loader.scss [hunk 1]
```diff
@@ -84,6 +84,24 @@ $-loader-spinner-path-speed: 1.5s;
...
+}
+
+.sage-loader__success-path {
+  stroke-dasharray: 150,200;
+  stroke-linecap: round;
+  stroke: sage-color(sage, 300);
+}
+
+.sage-loader__success-check {
+  transform: translate(76%, 84%);
```
Comment (anechol) [inline packages/sage-assets/lib/stylesheets/components/_loader.scss:99]: Would love to know what best practices are here when considering translating SVG paths.

**Review by philschanely — COMMENTED** [review]
Looking good! A few comments here. Also, could the React component also be updated please?

File: docs/lib/sage_rails/app/views/sage_components/_sage_loader.html.erb [hunk 2]
```diff
@@ -1,3 +1,7 @@
+<%
+  isSuccessful = component.type === "success" ? "Success" : "Loading"
```
[the remark this answers is not in the document]
Comment (anechol) [reply docs/lib/sage_rails/app/views/sage_components/_sage_loader.html.erb:2]: Yep, I agree. I was on the fence about this name.

**Comment by anechol** [comment]
React component is now updated.

**Review by philschanely — APPROVED** [review]
👍

**Review by pixelflips — APPROVED** [review]
LGTM! 👍🏼  Nice work!

**Pull request merged**